rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/AccessIdController.java

```diff
@@ -62,6 +62,47 @@ public AccessIdController(LdapClient ldapClient, TaskanaEngine taskanaEngine) {
     return response;
   }
 
+  /**
+   * This endpoint searches users for a provided name or Access Id. It will only search and return
+   * users and members of groups which are configured with the requested TASKANA role. This
+   * search will only work if the users in the configured LDAP have an attribute that shows their
+   * group memberships, e.g. "memberOf"
+   *
+   * @title Search for Access Id (users) in TASKANA user role
+   * @param nameOrAccessId the name or Access Id which should be searched for.
+   * @param role the role for which all users should be searched for
+   * @return a list of all found Access Ids (users)
+   * @throws InvalidArgumentException if the provided search for Access Id is shorter than the
+   *     configured one.
+   */
+  @GetMapping(path = RestEndpoints.URL_USER)
+  public ResponseEntity<List<AccessIdRepresentationModel>> searchUsersByNameOrAccessIdForRole(
```

Comment:
As we would like to replace this later on with LdapUserRepModel please create a ticket for this.